README.md

```diff
@@ -22,7 +22,7 @@ This style guide is based on the structure provided by @fat in [Medium's Style G
   - [ComponentName](#componentname)
   - [componentName--modifierName](#componentname--modifiername)
   - [componentName-descendantName](#componentname-descendantname)
-  - [componentName.is-stateOfComponent](#componentname.is-stateofcomponent)
+  - [componentName.is-stateOfComponent](#componentnameis-stateofcomponent)
```

Comment:
What's the use case for this (or the deleted version)? How does this differ from the `--modifierName`?